crates/bevy_ecs/src/world/mod.rs

```diff
@@ -988,8 +995,8 @@ impl World {
         match self.get_resource() {
             Some(x) => x,
             None => panic!(
-                "Requested resource {} does not exist in the `World`. 
-                Did you forget to add it using `app.insert_resource` / `app.init_resource`? 
+                "Requested resource {} does not exist in the `World`.
```

Comment:
These error messages changes are unrelated: we should strip them from this PR.

Reply:
Done, autoformatter snuck in :)